test/spec.dump

```diff
@@ -1,3 +1,29 @@
 {
-"annotations": {"io.container.manager": "libpod"}
+  "ociVersion": "1.0.2-dev",
+  "mounts": [
+    {
+      "destination": "/proc",
+      "type": "proc",
+      "source": "proc",
+      "options": [
```

Comment:
As you are only using `destination`, `type` and `source`, it seems unnecessary to include options.